pyface/undo/i_undo_manager.py

```diff
@@ -0,0 +1,63 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2008, Riverbank Computing Limited
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in enthought/LICENSE.txt and may be redistributed only
+# under the conditions described in the aforementioned license.  The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+# Thanks for using Enthought open source!
+#
+# Author: Riverbank Computing Limited
+# Description: <Enthought undo package component>
+# ------------------------------------------------------------------------------
+
+# Enthought library imports.
+from traits.api import Bool, Event, Instance, Int, Interface, Str
+
+
+class IUndoManager(Interface):
+    """The undo manager interface.  An undo manager is responsible for one or
+    more command stacks.  Typically an application would have a single undo
+    manager.
+    """
+
+    #### 'IUndoManager' interface #############################################
+
+    # This is the currently active command stack and may be None.  Typically it
+    # is set when some sort of editor becomes active.
+    active_stack = Instance("apptools.undo.api.ICommandStack")
```

Comment:
It looks like `IUndoManager` and `ICommandStack` depend on one another - which is why we can't import and use `ICommandStack` here. Can you add a comment here regarding the same?